core/src/apresolve.rs

```diff
@@ -12,24 +12,43 @@ pub struct AccessPoints {
     spclient: Vec<SocketAddress>,
 }
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Default)]
 pub struct ApResolveData {
     accesspoint: Vec<String>,
     dealer: Vec<String>,
     spclient: Vec<String>,
 }
 
-// These addresses probably do some geo-location based traffic management or at least DNS-based
-// load balancing. They are known to fail when the normal resolvers are up, so that's why they
-// should only be used as fallback.
-impl Default for ApResolveData {
-    fn default() -> Self {
+impl AccessPoints {
+    // These addresses probably do some geo-location based traffic management or at least DNS-based
+    // load balancing. They are known to fail when the normal resolvers are up, so that's why they
+    // should only be used as fallback.
+    fn fallback() -> Self {
         Self {
-            accesspoint: vec![String::from("ap.spotify.com:443")],
-            dealer: vec![String::from("dealer.spotify.com:443")],
-            spclient: vec![String::from("spclient.wg.spotify.com:443")],
+            accesspoint: vec![("ap.spotify.com".to_string(), 443)],
```

Comment:
Any specific reason you're changing this or just style preference?

Reply:
That was just habit

Reply:
I have no particular preference but generally let's try to separate style changes from actual changes, it keeps the history clean. Fine for now.

Reply:
The actual main change here was to store the APs directly as `(host, port)` / `SocketAddress` which avoids parsing the known-good data. So this was not a pure style change. 
That was also the cause of the weird filtering thing you commented on below